controller/sharding/sharding.go

```diff
@@ -39,6 +39,8 @@ func GetShardByID(id string, replicas int) int {
 	}
 }
 
+func GetClusterFilterForRoundRobbinAlgorithm() {}
```

Comment:
Are we planning to add logic for this in a separate PR?

Reply:
TBD